.travis.yml

```diff
@@ -14,9 +14,10 @@ branches:
   - master
   - development
 script:
-- git config --global user.email "arnab@binary.com"
-- git config --global user.name "Arnab Karmakar"
-- yarn deploy-gh-pages
+  - git config --global user.email "arnab@binary.com"
+  - git config --global user.name "Arnab Karmakar"
+  - yarn deploy-gh-pages
+  - yarn test-browserstack
```

Comment:
@4p00rv Why are we running test after deploy? How much time is it taking to run the tests? Are we not supposed to run the tests after build and before deploy? If tests fail, then deploy should fail too.